app/serializers/subject_serializer.rb

```diff
@@ -25,6 +25,10 @@ def already_seen
     !!(user_seen && user_seen.subject_ids.include?(@model.id))
   end
 
+  def seen_all_live_data
+    !!(user_seen && (user_seen.workflow.subject_ids.uniq.sort == user_seen.subject_ids.uniq.sort))
```

Comment:
Not sure that will take into account the live set of data (the linked subjects are fluid), we could use something like this https://github.com/zooniverse/Panoptes/blob/master/app/models/user.rb#L217